projects/aca-content/src/lib/components/favorite-libraries/favorite-libraries.component.ts

```diff
@@ -80,6 +95,10 @@ export class FavoriteLibrariesComponent extends PageComponent implements OnInit
     this.columns = this.extensions.documentListPresets.favoriteLibraries || [];
   }
 
+  onSearchVisibilityChange() {
```

Comment:
we have `extends PageComponent`, you can move `onSearchVisibilityChange` and corresponding properties to the base class to avoid code repetition and copy-pasting